eng/pipelines/ci.yml

```diff
@@ -102,14 +107,19 @@ jobs:
         condition: and(succeededOrFailed(), eq(variables['PythonVersion'], '3.10'))
         workingDirectory: $(Build.SourcesDirectory)/autorest.python/packages/autorest.python/test/azure/version-tolerant
 
-      - script: tox -e pyright
-        displayName: 'Pyright version-tolerant "azure" Tests - Python $(PythonVersion)'
-        condition: and(succeededOrFailed(), eq(variables['PythonVersion'], '3.10'))
```

Comment:
The deleted lines are duplicated items in current pipeline: https://github.com/Azure/autorest.python/blob/autorestv3/eng/pipelines/ci.yml#L100-L108